### DISCUSSION TITLE
Welcome to smtube Discussions!

### DISCUSSION THREAD

**Discussion by smplayer-dev** [smplayer-dev/smtube#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by touhamii** [comment]
how can i set 720p as default cuz im on an old pc

**Reply by Gitoffthelawn** [reply]
You'll want to create a new discussion for that.  This is just an "intro" discussion for people to say hello.

**Reply by touhamii** [reply]
it says : Ask questions you’re wondering about.

**Reply by Gitoffthelawn** [reply]
> it says : Ask questions you’re wondering about.

Correct.  And the way on GitHub to do that is by creating a discussion to ask your question.  Otherwise, you wind up with 1 huge thread with 1000's of questions that becomes useless over time.

**Comment by Gitoffthelawn** [comment]
I'll start by saying hello.  I honestly haven't used smtube yet, but I'm interested in learning what advantages it has over the YouTube web interface when it is combined with a good web browser extension.

I do hope smtube is extended to work on sites other than YouTube (if it isn't already), because I use Google-owned properties less and less every year.

**Reply by touhamii** [reply]
Hi, hope ur doing well the advantage for me is that i am looking to run youtube videos using the video player i want (mpv for instance)
but i need to be able to choose the quality for exemple in chrome when i install extension i can block 720p60 fps and this way i have kinda less drops in fps on my old pc (nvidia 7200 gs + pentium 4 3.2ghz) 
- overall the mpv is playing videos smoother than playing videos on browsers (for exemple youtube videos on chrome) that's the advantage 
i am running with 1gb ram on antiX 17.5 64bit

**Reply by Gitoffthelawn** [reply]
Thank you.  I hope you are doing well too.  The advantages you mention are very informative.  Thank you!
